install-timelord.sh

```diff
@@ -76,14 +76,23 @@ symlink_vdf_bench() {
 if [ "$(uname)" = "Linux" ] && type apt-get; then
 	UBUNTU_DEBIAN=true
 	echo "Found Ubuntu/Debian."
+
 elif [ "$(uname)" = "Linux" ] && type dnf || yum; then
 	RHEL_BASED=true
 	echo "Found RedHat."
+
+	if [ "$INSTALL_PYTHON_DEV" ]; then
+		yumcmd="sudo yum install "$PYTHON_VERSION"-devel gcc gcc-c++ gmp-devel libtool make autoconf automake openssl-devel libevent-devel boost-devel python3 cmake -y"
```

Comment:
The use of quotes around `$PYTHON_VERSION` actually unquotes this variable. If you wouldn't mind removing the quotes, I think that's my last piece of feedback before I hit the approve button :) Mostly to appease shellcheck but having it in the quotes is technically correct

Reply:
i have fixed this as requested.

Reply:
Thank you!